config/translate.go

```diff
@@ -57,7 +57,7 @@ func TranslateFromV1(old v1.Config) types.Config {
 
 		for _, oldPartition := range oldDisk.Partitions {
 			disk.Partitions = append(disk.Partitions, types.Partition{
-				Label:    string(oldPartition.Label),
+				Label:    func(s string) *string { return &s }(string(oldPartition.Label)),
```

Comment:
Use `internal/util:StringToPtr`

https://github.com/coreos/ignition/blob/master/internal/util/util.go#L25

Reply:
Do we want to have the config section depend on something in internal? Then things that consume the config package will also consume internal.

Reply:
Not particularly. Maybe we should move it to a top level util directory, thoughts @dgonyeo?

Reply:
I can't say I see the harm in consumers of the `config` package becoming consumers of the `internal` package. Based on the golang `internal` package semantics, they won't be able to use anything in it, so the fact that it's going into their `vendor` directories seems to me like an implementation detail.

On the other hand the fact that we're considering adding a dependency to an entirely different package in here, that we originally didn't intend for users to have to include in their projects, just because golang's type system is awful, seems entirely absurd.